packages/core/src/model/model.ts

```diff
@@ -1,11 +1,6 @@
+import { format } from 'date-fns'
```

Comment:
Recommend this for better bundling with date-fns@3...
```
import { format } from 'date-fns/format'
```

Reply:
@RiyaJethwa could you pls take a look at this as well?